CMake/sitkUseGTest.cmake

```diff
@@ -33,7 +33,9 @@ function(_sitk_gtest_use_gtest_source)
 
   set(BUILD_GTEST                 ON )
   set(BUILD_GMOCK                 OFF)
-
+  if ("${WIN32}")
+      set(gtest_disable_pthreads      ON CACHE INTERNAL "Disable use of pthreads in gtest")
+  endif()
```

Comment:
I am noting that CMake 3.1, does not define `MINGW` but [CMake 3.2 does](https://cmake.org/cmake/help/v3.2/variable/MINGW.html).